picard/script.py

```diff
@@ -894,6 +894,24 @@ def iswbound(char):
     return "Zs" == category or "Sk" == category or "P" == category[0]
 
 
+def func_is_audio(parser):
+    """Returns true, if the file processed is an audio file."""
+    if func_is_video(parser) == "1":
+        return ""
+    else:
+        return "1"
+
+
+VIDEO_EXTENSIONS = ['m4v', 'wmv', 'ogv', 'oggtheora']
```

Comment:
Since it is exported (at least to tests), I'd rather move this constant at top of file and since it isn't meant to be modified (is it?) I think we can use a tuple instead of a list.

Reply:
Actually I think we should handle this differently and files should set the `~video` tag themselves. This moves the responsibility of handling this to the specific formats. This constant was a workaround needed by the plugin. I'll look into this.